lib/__tests__/Configuration-test.js

```diff
@@ -363,14 +363,97 @@ describe('Configuration', () => {
           { environments: ['node', 'meteor'] });
       });
 
-      it('returns node core modules', () => {
-        const coreModules = new Configuration().environmentCoreModules();
+      it('returns core modules from all environments', () => {
+        const coreModules = new Configuration().get('coreModules');
         expect(coreModules).toContain('child_process');
         expect(coreModules).toContain('meteor/check');
       });
     });
   });
 
+  describe('#get aliases', () => {
+    beforeEach(() => {
+      // TODO: replace the array-style configuration here when we have an
+      // environment that specify aliases.
+      FileUtils.__setFile(
+        path.join(process.cwd(), '.importjs.json'),
+        [
+          {
+            aliases: {
+              foo: 'foosball',
+            },
+          },
+          {
+            aliases: {
+              bar: 'barsketball',
+            },
+          },
+          {
+            aliases: () => ({
+              baz: 'bazeball',
+            }),
+          },
+        ]);
+    });
+
+    it('merges all aliases into a single object', () => {
+      expect(new Configuration().get('aliases')).toEqual({
+        foo: 'foosball',
+        bar: 'barsketball',
+        baz: 'bazeball',
+      });
+    });
+  });
+
+  describe('#get namedExports', () => {
+    beforeEach(() => {
+      // TODO: replace the array-style configuration here when we have an
+      // environment that specify namedExports.
+      FileUtils.__setFile(
+        path.join(process.cwd(), '.importjs.json'),
+        [
+          {
+            namedExports: {
+              foo: ['foosball'],
+            },
+          },
+          {
+            namedExports: () => ({
+              bar: ['barsketball'],
```

Comment:
Yes. I thought about adding support for merging those arrays but I don't know if it's worth it.